### PR TITLE
HTSeq: init at 0.11.2

### PR DESCRIPTION
**Pull request by unode**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/HTSeq/default.nix [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "65c4c13968506c7df92e97124df96fdd041c4476c12a548d67350ba8b436bcfc";
+  };
+
+  buildInputs = [ cython numpy pysam ];
+  propagatedBuildInputs = [ numpy pysam matplotlib ];
+
+  meta = with stdenv.lib; {
```
Comment (Lassulus) [inline pkgs/development/python-modules/HTSeq/default.nix]: a description would be very nice

File: pkgs/development/python-modules/HTSeq/default.nix [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{ stdenv, buildPythonPackage, fetchPypi, cython, numpy, pysam, matplotlib
+}:
+buildPythonPackage rec {
+  version = "0.11.2";
+  pname = "HTSeq";
+  name = "${pname}-${version}";
```
Comment (Lassulus) [inline pkgs/development/python-modules/HTSeq/default.nix]: is this line needed?

File: pkgs/development/python-modules/HTSeq/default.nix (hunk 1 above)
Comment (unode) [reply pkgs/development/python-modules/HTSeq/default.nix]: Added

File: pkgs/development/python-modules/HTSeq/default.nix (hunk 2 above)
Comment (unode) [reply pkgs/development/python-modules/HTSeq/default.nix]: Removed

**Review by Lassulus — APPROVED** [review]
tested with nix-review, started python, imported HTSeq und played around a little bit

**Comment by unode** [comment]
Thanks for the review!
Cheers

**Pull request merged**